### DISCUSSION TITLE
Give the file picker a window title?

### DISCUSSION THREAD

**Discussion by rieje** [mifi/lossless-cut#1954] [Q&A] [answered]
I'm using a tiling window manager (Sway) and when I select a file using the file picker, it doesn't have a window name. I would like it to have a window title so that it can be matched with a window rule and make it by default floating, for example. Is this possible? E.g. Window title would be something like "File manager" then a window rule can be: app_id=losslesscut, title="File manager".

P.S. Instead of defaulting to Recent, is it possible to remember previously opened folder (or at least set path to a folder)? I only import a file once for the most part so Recent is not useful and having to select the folder to open from every time is inconvenient.

**Comment by mifi** [comment, marked as answer]
i will add a title "Open file"

**Reply by enoryw** [reply]
Hi, I was also waiting for this feature to be able to set default file picker window sizes for tiling window manager but it looks like it's either not working or not all similar windows are supported, e.g. on the latest version, File > Import project > ... opens the picker with no window title. I'm not a programmer but this is the relevant commit: https://github.com/mifi/lossless-cut/commit/88fbcc0129f3e360557f70a5d45d115803ec9e7a

**Reply by mifi** [reply]
did you try the nightly build?

**Reply by enoryw** [reply]
Just tested on nightly build and experiencing the same thing. The 'Open file' title only applies to picker from "File > Open".

**Reply by mifi** [reply]
Gotcha, I will add a title to the other dialogs too
